src/openrct2/windows/TileInspector.cpp

```diff
@@ -1874,15 +1874,15 @@ static void window_tile_inspector_paint(rct_window *w, rct_drawpixelinfo *dpi)
         {
             // Details
             // Type
-            sint16 largeSceneryType = tileElement->properties.scenerymultiple.type & 0x03FF;
+            sint16 largeSceneryType = scenery_large_get_type(tileElement);
             gfx_draw_string_left(dpi, STR_TILE_INSPECTOR_LARGE_SCENERY_TYPE, &largeSceneryType, COLOUR_DARK_GREEN, x, y);
 
             // Part ID
-            sint16 pieceID = (tileElement->properties.scenerymultiple.type & 0xFC00) >> 10;
+            sint16 pieceID = scenery_large_get_sequence(tileElement);
```

Comment:
We should rename everything to piece id our sequence and adjust the code to suit

Reply:
What do you mean?